testhelpers/docker_registry.go

```diff
@@ -68,3 +123,52 @@ func (registry *DockerRegistry) Stop(t *testing.T) {
 		DockerCli(t).ContainerRemove(context.TODO(), registry.Name, types.ContainerRemoveOptions{Force: true})
 	}
 }
+
+func (registry *DockerRegistry) RepoName(name string) string {
```

Comment:
```suggestion
func (r *DockerRegistry) RepoName(name string) string {
```
Not sure why this and others had `registry` as a receiver instead of `r` but we should probably fix it while we are looking at this.